dist/common/systemd/scylla-fstrim.timer

```diff
@@ -1,7 +1,6 @@
 [Unit]
 Description=Run Scylla fstrim weekly
 After=scylla-server.service
-BindsTo=scylla-server.service
```

Comment:
Is `After` enough to get the timer to start? Or maybe er enable it explicitly in scylla_setup with `systemctl enable`?

Reply:
scylla-fstrim.timer should be disabled until we enable it by scylla_fstrim_setup (it runs `systemctl enable`).
And I guess we don't need `After=` anymore, because it was to start scylla-fstrim only after scylla-server, but now we allow running the service without scylla-server, so we don't need to setting service order.